### PR TITLE
Clean up scroll timeouts

### PR DESCRIPTION
**Pull request by jbalboni**
## Description
I think some uncleared timeouts and non-overridden scrolling delays are causing some of our flaky tests. This attempts to fix those.

## Testing done
Unit testing

## Acceptance criteria
- [ ] Tests pass

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/burials/tests/helpers.unit.spec.jsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ import {
 describe('Burials helpers', () => {
   describe('submit', () => {
     beforeEach(() => {
-      window.VetsGov = { pollTimeout: 1 };
+      window.VetsGov.pollTimeout = 1;
```
Comment (jbalboni) [inline src/applications/burials/tests/helpers.unit.spec.jsx:15]: There's one place in the app that uses the scroll params from `VetsGov`, and these pollTimeout lines were previously overwriting the code we put in place to set the scroll delays to 0.

File: src/platform/forms-system/src/js/fields/ArrayField.jsx [hunk 2]
```diff
@@ -105,7 +110,9 @@ export default class ArrayField extends React.Component {
   }
 
   scrollToRow(id) {
-    setTimeout(() => {
+    this.rowTimeout = setTimeout(() => {
+      // eslint-disable-next-line no-console
```
[the remark this answers is not in the document]
Comment (jbalboni) [reply src/platform/forms-system/src/js/fields/ArrayField.jsx:114]: It's a draft PR, calm down

**Review by rianfowler — APPROVED** [review]
This seems fine. `window.Forms` seems overly generic but no strong feelings on that.

**Comment by jbalboni** [comment]
@rianfowler `window.Forms` is being used already, I was just matching it.

**Pull request closed without merging**